packages/terra-application/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+* Breaking
+  * Dropped v2-v4 react-intl support
+
+* Removed
+  * References to terra-base and moved terra-base logic to terra-application-base
+
+* Added
+  * Added i18n logic
```

Comment:
Agian, I don't think that this needs to be included in the changelog, we didn't really add i18n logic so much as moved where it was being implemented and how. I think the `Breaking` changes are adequate for consumers here.